docs/docs/charts/scatter.mdx

```diff
@@ -50,7 +50,7 @@ function example() {
 ## Dataset Properties
 
 The scatter chart supports all of the same properties as the [line chart](./charts/line.mdx#dataset-properties).
-By default, the scatter chart will override the showLine property of the line chart to `false`.
+By default, the scatter chart will override the showLine property of the line chart to `false` and the type for the data array has changed to `object[]`.
```

Comment:
Scatter supports same data formats, only the index scale is `linear`. So labels array needs to be numbers (or convertable to number), if that format is used. Same with object data, the keys need to be convertible to numbers.

https://codepen.io/kurkle/pen/ZEBaaZK